control-center/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>io.confluent.control-center-images</groupId>
         <artifactId>control-center-images-parent</artifactId>
-        <version>7.0.15-0</version>
+        <version>7.8.0-0</version>
```

Comment:
Hi @VanamalaNarasimhaBhargav

I checked https://github.com/confluentinc/control-center-images/pull/67/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L41

I dont see image version getting changed to 7.8 in the above commit. Not sure I follow why are we updating image version to 7.8, specially in branch 7.0.x

Reply:
@nagarwal01 This PR https://github.com/confluentinc/control-center-images/pull/67 was raised by Jenkins by service bot  and the commit [Porting changes to 7.0.x](https://github.com/confluentinc/control-center-images/commit/d8dacf1d9112e4f93e2a5c0bc66dae998b2109b5) was made by me manually into the same PR.

To cross verify this check the last commit in [#67](https://github.com/confluentinc/control-center-images/pull/67/commits) where the change you mentioned above will be visible.

To verify further check the [last 4 commits](https://github.com/confluentinc/control-center-images/commits/7.0.x/pom.xml) on the branch 7.0.x on the file pom.xml which should make things clear.

> Not sure I follow why are we updating image version to 7.8, specially in branch 7.0.x

This version was updated to latest by [Jenkins bot ](https://github.com/confluentinc/control-center-images/commit/38f3e2e9acc45bca5e1d4fbf23364fe2ebbb24af) on all branches. Plan here is to revert the commit in 7.0.x & propogate it to other branches using pint merge.